lib/elixir/lib/task/supervised.ex

```diff
@@ -189,20 +189,46 @@ defmodule Task.Supervised do
 
   ## Stream
 
-  def stream(enumerable, acc, reducer, callers, mfa, options, spawn) do
-    next = &Enumerable.reduce(enumerable, &1, fn x, acc -> {:suspend, [x | acc]} end)
-    max_concurrency = Keyword.get(options, :max_concurrency, System.schedulers_online())
+  def validate_stream_options!(options) do
```

Comment:
Note: this function is called immediately when building the stream, not when evaluating it.